### PR TITLE
some fliers, notes, news

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary

SUMMARY: Content "adds a handful of new fliers and news reports"

#### Purpose of change

I'm trying my hand at writing textual ephemera for the game.  These snippets build upon existing lore as found in newspapers, lab reports, and fliers.

#### Describe the solution

One new survivor note:

_...bioluminescent, 5' diameter⋯  appears to be an immense disembodied eyeball, capable of flight and [illegible] thought_

Six fliers:

1. _This ancient missing persons poster is faded and torn.  It features a photo of a stern looking woman in her early fifties.  It reads "Missing Adult - Prof. Amy Takatoshi - Last Seen Thursday, July 12 - Please contact the Boston Police Missing Persons Unit."_
2. _Join the Concerned Consumers of Foodpeople.  Join the fight to⋯  -Stop the neutralization of cultural icons.  -Endow Foodperson with vitals that befitting heroics superb.  -Protect RU-238 and other FRUIT flavors from the threat of prohibition.  -Inspire future consumers of the superhero genre._
3. _This is an ad for a job fair.  \"Summer Internship & Job Fair!  Open to students, alumni, and the public.  Both free and unpaid internships available.  Live and virtual representatives from exciting organizations like: Actem Labs, Cuppatech, MindStone, Edison, NourishMart, and more!\"_
4. _This scrap of paper features a faded ad for Cascade Cola.  \"All the sweet you can stand.  Don't just drink it; let it cascade into you.\"_
5. _This promotional palm card is emblazoned "DigiMart has E-Ink" in a garish purple typeface.  "E-Ink Tablets come packed with octo-core cpu, veripixel lenses camera, neoTouch screen, DRM locket, InnerVision ready near field, a free datavault storage plan, wi-fi, and more!"_
6. _What you've got here is an old coupon insert for Marigold Market:\n\n - BOGO!  Cascade Cola six-packs!\n - ATOMIC POWER THIRST!  Now with 20% more FRUIT RU-238!\n - $2 MEAT! Aunt Janice's Pickled Meat now just $2.49!\n - 75% OFF! DaiZoom bars by SoyPelusa._
7. _From Your Devoted Sister_  - see comments or files changed for content

Four news reports:

1. One year old: _MENDING FENCES Growing international tensions have prompted President Oswald to call for unprecedented funding increases for security revitalization.  The president is calling for new \"precision micro\" military installations to be built across the nation, with thousands to be constructed in New England alone.  These come alongside similarly ambitious budget increases for FEMA and scientific research.  A coalition of civil rights orgs, neighborhood associations, and environmental advocates have denounced the initiative._
2. Weeks old: _COPS IN CRISIS  Law enforcement officials report increasing numbers of officers are calling in sick, have resigned, or are just not showing up to duty.  A lack of sufficient officers exacerbates the dangers on duty police face.  In some cases officers are refusing to respond to dispatches in areas with high levels of violence.  A spokesperson for the Worcester County Sheriff's Office confirmed "It is no longer feasible for us to jeopardize officer lives in some circumstances.  Instead we can continue to rely upon our state of the art AI powered robotic units."_
3. One year old: _BIGGER BEES.  State biologists are unable to explain the emergence of bees that are two and three times normal size.  Authorities say that bees nearly the size of grapes are being reported across the state.  As of this time, there have been no reports of increased aggression or other behavioral changes in the oversized insects.  \"I'm glad to see the resurgence in the bee population,\" said Dr. Melissus \"Mel\" Versed, an entomologist at Tufts University.  He added, \"while Anthophila this large are unprecedented in New England, there is nothing particularly harmful about the phenomena.  Think of the honey yields!\"_
4. Many years old (by necessity) _GENERAL PROMOTED ONBOARD OLD IRONSIDES  The USS Constitution was the site of a promotion ceremony held by the Massachusetts National Guard at the Charlestown Navy Yard.  Col. Ariel Dabrowski was promoted to the rank of brigadier general in the Massachusetts National Guard in front of a crowd massed on the deck of the historic ship.  General Carlsberg officiated the promotion ceremony, and Maj. Gen. Michael Baker administered the oath of office._

#### Describe alternatives you've considered

President Oswald wasn't previously gendered.  I considered using the "she" pronoun, but it was pointed out that having the world end during our first female president's term is not a good look.  So Oswald is a dude.

I don't have an knowledge of or experience with military protocol, so any content I write on this subject might be hamfisted, or just plain foolish.

#### Testing

This new content does need additional copy editing, hence WIP.

I've looked up all of the named entities to ensure continuity with existing lore, and in some cases double checked with other contributors.

### CODE REVIEW

File: data/json/snippets/newspapers.json [hunk 1]
```diff
@@ -350,6 +362,10 @@
       {
         "id": "weeks_old_news_17",
         "text": "LOCAL EVACUATION ORDERS.  FEMA has announced small-scale evacuations of select centers where rioting is most intense.  \"This is a preventive measure,\" Spokesperson Linda Garrison told our correspondent.  \"We expect to have people home in a few days, but it's best we play it safe until then.\""
+      },
+      {
+        "id": "weeks_old_news_18",
+        "text": "COPS IN CRISIS.  Law enforcement officials report increasing numbers of officers are calling in sick, have resigned, or are just not showing up to duty.  A lack of sufficient officers exacerbates the dangers on duty police face.  In some cases officers are refusing to respond to dispatches in areas with high levels of violence.  A spokesperson for the Worcester County Sheriff's Office confirmed \"It is no longer feasible for us to jeopardize officer lives in some circumstances.  Instead we can continue to rely upon our state of the art AI powered robotic units.\""
```
Comment (geophree) [inline data/json/snippets/newspapers.json:368]: `A lack of sufficient officers` -> `A lack of sufficient personnel`
`on duty police` -> `on-duty police`
`state of the art AI powered robotic units` -> `state-of-the-art AI-powered robotic units`

**Comment by geophree** [comment]
For: `This scrap of paper features a faded ad for Cascade Cola. "All the sweet you can stand. Don't just drink it; drink it down."`, maybe instead of `drink it down` something like `let it cascade into you`?

**Comment by Fris0uman** [comment]
Maybe make `Dr. Melissus "Mel" Versed` a beekeeper instead, because bees the size of a golf ball would be very concerning to a researcher X).

**Comment by Jerimee** [comment]
> Maybe make `Dr. Melissus "Mel" Versed` a beekeeper instead, because bees the size of a golf ball would be very concerning to a researcher X).
> Or mention bigger bees but not golf ball size yet.

Dr. Versed ends up becoming a XEDRA researcher - he is in the lab snippets. So maybe not the most conscientious scientist.

I couldn't figure out a way to write it in, but there is actually a IRL bee with 3 inch wing span: Wallace's Giant Bee.

What should I change it to? The size of a walnut? The size of a quarter? Size of a grape?

**Comment by Fris0uman** [comment]
You can leave it as ` emergence of bees that are two and three times normal size.` But if it's actually disinformation from XEDRA people maybe just leave it as is then ^^.

**Comment by ZhilkinSerg** [comment]
```
src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 316:173: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

      {

        "id": "flier_78",

        "text": "This ancient missing persons poster is faded and torn.  It features a photo of a stern looking woman in her early fifties.  It reads \"Missing Adult - Prof.

                                                                                                                                                                            ^

                                                                                                                                                                              Amy Takatoshi - Last Seen Thursday, July 12 - Please contact the Boston Police Missing Persons Unit.\""

      },

      {

22:01:19.362 ERROR : src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 328:187: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

      {

        "id": "flier_81",

        "text": "What you've got here is an old coupon insert for Marigold Market:\n\n - BOGO!  Cascade Cola six-packs!\n - ATOMIC POWER THIRST!  Now with 20% more FRUIT RU-238!\n - $2 ME

                                                                                                                                                                                          ^

                                                                                                                                                                                           AT! Aunt Janice's Pickled Meat now just $2.49!\n - 75% OFF! DaiZoom bars by SoyPelusa."

      }

    ]

22:01:19.362 ERROR : src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 328:242: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

        "text": "What you've got here is an old coupon insert for Marigold Market:\n\n - BOGO!  Cascade Cola six-packs!\n - ATOMIC POWER THIRST!  Now with 20% more FRUIT RU-238!\n - $2 MEAT! Aunt Janice's Pickled Meat now just $2.49!\n - 75%

                                                                                                                                                                                                                                                ^

                                                                                                                                                                                                                                                 OFF! DaiZoom bars by SoyPelusa."

      }

    ]

22:01:19.369 ERROR : src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 76:193: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

      {

        "id": "many_years_old_news_18",

        "text": "GENERAL PROMOTED ONBOARD OLD IRONSIDES.  The USS Constitution was the site of a promotion ceremony held by the Massachusetts National Guard at the Charlestown Navy Yard.  Col.

                                                                                                                                                                                                ^

                                                                                                                                                                                                 Ariel Dabrowski was promoted to the rank of brigadier general in the Massachusetts National Guard in front of a crowd massed on the deck of the historic ship.  General Carlsberg officiated the promotion ceremony, and Maj. Gen. Michael Baker

22:01:19.369 ERROR : src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 76:415: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

wn Navy Yard.  Col. Ariel Dabrowski was promoted to the rank of brigadier general in the Massachusetts National Guard in front of a crowd massed on the deck of the historic ship.  General Carlsberg officiated the promotion ceremony, and Maj.

                                                                                                                                                                                                                                                ^

                                                                                                                                                                                                                                                 Gen. Michael Baker administered the oath of office."

      }

    ]

22:01:19.370 ERROR : src/translations.cpp:461 [auto translation::deserialize(JsonIn &)::(anonymous class)::operator()(const std::string &, const int) const] 

line 76:420: insufficient spaces at this location.  2 required, but only 1 found.

    Suggested fix: insert " "

    At the following position (marked with caret)

vy Yard.  Col. Ariel Dabrowski was promoted to the rank of brigadier general in the Massachusetts National Guard in front of a crowd massed on the deck of the historic ship.  General Carlsberg officiated the promotion ceremony, and Maj. Gen.

                                                                                                                                                                                                                                                ^

                                                                                                                                                                                                                                                 Michael Baker administered the oath of office."

      }

    ]
```

**Comment by Jerimee** [comment]
still need to //NOLINT format GENERAL PROMOTED ONBOARD OLD IRONSIDES

**Comment by Jerimee** [comment]
adding a new flier_71:

```
Dear Brother,

I hope you get this letter before it is too late.  And I hope you understand I am sending this letter out of respect and love and, I'm not ashamed to admit it, fear.  Fear for you and fear for myself.  There is a reason why people are stocking up on guns and flour.

People are saying the rioters are merely insane, or under the influence of that new drug, or fever dream sleep walking.  I don't know what's true and I don't care.  They say almost the exact same thing about the eye-witnesses; maybe we're all fever dreaming now.

This isn't going to be delayed, let alone solved, by rifles or protein rations or military robots authorized to kill.  The writing is on the wall.  I'm not saying that my way is good or holy, and I love and respect you no matter what, but I hope and I pray that you'll join me.

There isn't much time to decide.  Soon we might be the last folks on earth.  You know where to go. 

- Your Devoted Sister
```

**Pull request merged**